FlightPlanProject/Tools/UI/FPStyles.cs

```diff
@@ -0,0 +1,331 @@
+
```

Comment:
Contains all the UI styles. very long class....

It's not exactly the same as K2D2, contains more style for box and scroll bar for instance